technical-reference/reference-technical-articles/technical-doc-token-converter.md

```diff
@@ -0,0 +1,226 @@
+
+Examples:
+
+- Invocation 1: `getAmountIn(40 * 10^18, address(XVS), address(USDC))`
+    - The configuration enables for `tokenAddressIn / tokenAddressOut`: XVS/USDC
+    - Current liquidity: 100 USDC
+    - Current conversion rate, given by the oracle: 1 XVS = 5 USDC
+    - Configured discount: 0%
+    - Response:
+        - `amountInMantissa`: 8 * 10^18 (XVS)
+        - `amountConvertedMantissa`: 40 * 10^18 (USDC), equal to the `amountOutMantissa` param
+- Invocation 2: `getAmountIn(120 * 10^18, address(XVS), address(USDC))`
+    - The configuration enables for `tokenAddressIn / tokenAddressOut`: XVS/USDC
+    - Current liquidity: 100 USDC
+    - Current conversion rate, given by the oracle: 1 XVS = 5 USDC
+    - Configured discount: 0%
+    - Response:
+        - `amountInMantissa`: 20 * 10^18 (XVS)
+        - `amountConvertedMantissa`: 100 * 10^18 (USDC), different from the `amountOutMantissa` param because there is not enough liquidity
+
+### Conversions fixing the amount in
+
+Function `convertExactTokens(uint256 amountInMantissa, uint256 amountOutMinMantissa, address tokenAddressIn, address tokenAddressOut, address to)`, where:
+
+- `amountInMantissa`: the number of tokens (`tokensAddressIn`) the sender wants to convert
+    - It’s defined in the mantissa of the `tokenAddressIn`
+    - Examples:
+        - 1 [TRX](https://bscscan.com/address/0xce7de646e7208a4ef112cb6ed5038fa6cc6b12e3) (6 decimals in BSC), will be 1000000
+        - 1 [USDC](https://bscscan.com/address/0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d) (18 decimals in BSC), will be 1000000000000000000
+- `amountOutMinMantissa`: the minimum amount of `tokenAddressOut` tokens the sender accepts to receive in the conversion. It’s defined in the mantissa of the `tokenAddressOut`.
+    - If the amount that would be finally transferred to `to` is less than `amountOutMinMantissa`, the transaction is reverted
+- `tokenAddressIn`: the address of the token provided by the sender to perform the conversion. For instance, in the `XVSVaultConverter` and in the `RiskFundConverter` there will be configurations to accept XVS and USDT token addresses respectively
+- `tokenAddressOut`: the address of the token the sender wants to receive with the execution of the conversion
+- `to`: the address where the tokens of `tokenAddressOut` should be sent
+
+The received tokens (`amountInMantissa`) are transferred, in the same transaction, to the `destination` address. The `destination` address is configured as a storage attribute in the `AbstractTokenConverter` contract.
+
+If the received amount is less than `amountInMantissa`, or the transferred out amount is less than the expected one, the transaction is reverted, because we don’t support tokens with fees on transferring in this function.
```

Comment:
```suggestion
If the received amount is less than `amountInMantissa` or the transferred out amount is less than the expected one, the transaction is reverted because we don’t support tokens with fees on transferring in this function.
```